docs/GettingStartedDocs/VisualStudioLinux.md

```diff
@@ -39,6 +41,9 @@ On the Linux build machine, or after opening an ssh session into the VM:
   prior to the Install section, which should work with either 16.04 or 18.04 even though
   the page only mentions 16.04.
 
+ - If you would like to install the Open Enclave Host-Verify SDK instead, see [installation instructions for Ubuntu 16.04](https://github.com/microsoft/openenclave/blob/master/docs/GettingStartedDocs/install_host_verify_Ubuntu_16.04.md)
```

Comment:
I suggest moving this paragraph to above line 10 and have its own short section to redirect the reader to host-verify SDK link if they just want to build evidence verification app.

Building app against host-verify SDK is like developing regular app. It's irrelevant to SGX or enclave.